sp_mock/frontend/src/views/LoginRegister.vue

```diff
@@ -78,11 +78,32 @@ const loginWithLayer8Popup = async () => {
   const response = await layer8.fetch("http://localhost:8000/api/login/layer8/auth")
   const data = await response.json()
 
+  //alert(data.authURL)
   // create opener window
   const popup = window.open(data.authURL, "Login with Layer8", "width=600,height=600");
+ 
   window.addEventListener("message", async (event) => {
-    popup.close();
-    window.location.href = event.data.url;
+    if(event.data.redr){
```

Comment:
This one is just great! I don't know why didn't I thought this way to solve the no reload issue!